packages/manager/apps/pci-object-storage/src/components/StatusBadge.tsx

```diff
@@ -0,0 +1,35 @@
+import { OdsBadge } from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import { TOdsBadgeColor } from './types';
+
+export type TStatusBadgeProps = {
+  status: string;
+  translationKey: string;
```

Comment:
component should be agnostic from translation, please give directly the translated label through props